interceptors/src/unit-test/java/uk/gov/moj/sdt/interceptors/out/ServiceRequestOutboundInterceptorTest.java

```diff
@@ -73,24 +54,25 @@ void testHandleMessage() throws IOException {
         SdtContext.getContext().setServiceRequestId(1L);
         String data = "<xml>content</xml>";
         final SoapMessage soapMessage = getDummySoapMessageWithCachedOutputStream(data);
-        final ServiceRequestOutboundInterceptor serviceRequestOutboundInterceptor = new ServiceRequestOutboundInterceptor(mockServiceRequestDao);
+        final ServiceRequestOutboundInterceptor serviceRequestOutboundInterceptor = new ServiceRequestOutboundInterceptor(requestDaoService);
         final ServiceRequest serviceRequest = new ServiceRequest();
         when(mockServiceRequestDao.fetch(ServiceRequest.class, 1L)).thenReturn(serviceRequest);
         mockServiceRequestDao.persist(serviceRequest);
```

Comment:
This is actually calling the persist() method on mockServiceRequestDao. It's unlikely that the test needs to do this and it's probably a hang over from Easy Mock. This may need to be changed to verify(mockServiceRequestDao).persist(serviceRequest).